mer_verify_kernel_config

```diff
@@ -129,6 +129,7 @@ CONFIG_ANDROID_PARANOID_NETWORK	y,n       # Since Android 5 on some devices this
 CONFIG_AUDIT			n,!	# This will disable SELinux! That's ok, because hybris adaptations must not have SELinux, but if your device needs its support in kernel, set AUDIT=y and SELINUX_BOOTPARAM=y. Then disable them via kernel cmdline: audit=0 selinux=0. You can also leave audit enabled, if you don't plan to use systemd's containers: http://cgit.freedesktop.org/systemd/systemd/commit/README?id=77b6e19458f37cfde127ec6aa9494c0ac45ad890
 CONFIG_AUTOFS4_FS		y,m,!	# systemd (optional): http://cgit.freedesktop.org/systemd/systemd/commit/README?id=713bc0cfa477ca1df8769041cb3dbc83c10eace2
 CONFIG_BRIDGE			y,m,!   # connman (optional): support tethering, http://git.kernel.org/cgit/network/connman/connman.git/commit/README?id=19fe7cad485afa6a7a5cc4aa75615ce8b7b8d376
+CONFIG_CPUSETS			y,m,!	# lxc (optional): required to run lxc containers used by Halium, LuneOS etc. 
```

Comment:
LXC is also used for other things like Android app support in Sailfish OS, so also that should have a more generic description.

Reply:
Ping, could you update the descriptions?